.github/workflows/tiledb-go.yml

```diff
@@ -37,7 +37,7 @@ jobs:
     strategy:
       matrix:
         # Will be checking following versions
-        go: ["1.22", "1.23"]
+        go: ["1.23", "1.24"]
```

Comment:
Golang 1.25 is not released yet, so I stepped back to 1.23 instead testing 1.24 alone. If we don't need to look backwards like this I can remove and just test against 1.24.

Reply:
This can't actually run on Go 1.23 because of the version declaration in `go.mod`. I guess CI still runs it in 1.24?